### ISSUE TITLE
Update setting for 0/editorial-team-prep label

### ISSUE THREAD

**Issue by mpadge** [ropensci-org/buffy#102]
@lwasser brought to our attention that the `0/editorial-team-prep` label is currently not automatically added. It is removed here:
https://github.com/ropensci-org/buffy/blob/044d630b554a7963aa2a1d8febe9422edd5643bd/config/settings-production.yml#L103-L110
But never automatically added. We can easily automatically add on initial submission if a pre-sub enquiry can be identified. The question we'd have to resolve would be what to do with initial *full* submissions? Should we add this label straight away? There is currently no other viable point at which we could automatically do this. Thoughts @noamross @maelle @lwasser?

**Comment by lwasser** [comment]
yes, we current start with a 1/editor-checks but have been thinking about adding a 0/editorial-team-prep type label for new submissions that haven't been "Assigned" yet!

as such i agree with @noamross above 

This is what a new submission. looks like for us. but really it should be a `0/editorial-team-prep` or  `0/review-triage` type of label prior to checks as that is it's current state.

<img width="622" alt="Screenshot 2024-01-05 at 12 16 36 PM" src="https://github.com/ropensci-org/buffy/assets/7649194/b732b721-9042-40c9-b30c-129692dda2b6">
